src/svgcontext.ts

```diff
@@ -113,18 +167,21 @@ export class SVGContext implements RenderContext {
     };
 
     this.state_stack = [];
-
-    // Test for Internet Explorer
-    this.iePolyfill();
   }
 
+  create(svgElementType: 'g'): SVGGElement;
+  create(svgElementType: 'path'): SVGPathElement;
+  create(svgElementType: 'rect'): SVGRectElement;
+  create(svgElementType: 'svg'): SVGSVGElement;
+  create(svgElementType: 'text'): SVGTextElement;
+  create(svgElementType: string): SVGElement;
   create(svgElementType: string): SVGElement {
     return document.createElementNS(SVG_NS, svgElementType);
   }
 
   // Allow grouping elements in containers for interactivity.
   openGroup(cls: string, id?: string, attrs?: { pointerBBox: boolean }): SVGGElement {
-    const group: SVGGElement = this.create('g') as SVGGElement;
+    const group: SVGGElement = this.create('g');
```

Comment:
Should we remove `: SVGGElement`? TypeScript can infer the type, and the variable name is pretty descriptive.

We can keep explicit local variable types when there is potential for confusion.

Your call. Either way works for me!

Reply:
Done